server/publications/collections/accounts.js

```diff
@@ -45,6 +45,22 @@ Meteor.publish("Accounts", function (userId) {
 });
 
 /**
+ * Single account
+ * @params {String} userId -  id of user to find
+ */
+Meteor.publish("Accounts.single", function (userId) {
```

Comment:
Also, the `Accounts` global subscription is called `Account` for some reason, so we couldn't follow the `Product/Products` pattern (if we even should)